examples/tests/config.yaml

```diff
@@ -0,0 +1,2 @@
+global:
+    debugger_port: 4999
```

Comment:
ah ha, this explains #231

Reply:
Exactly :D.

Couldn't get earth within earth via `WITH DOCKER` working just yet, however. It seems that `docker run --privileged` does not work (we need that to start buildkitd). Anyway, most people don't need that so we should be ok for now.